assets/src/scripts/selectors/time-series-selector.js

```diff
@@ -122,9 +149,11 @@ export const getRequestTimeRange = memoize((tsKey, period, parmCd) => createSele
                 end: notes.requestDT
             };
         } else {
+            let intervalStart = notes['filter:timeRange'].interval.start;
```

Comment:
I'd like to a see a test case in time-series-selector.spec.js for this additional functionality.